study-builder/tenants/testboston/emails/change_password.html

```diff
@@ -1,106 +1,193 @@
-<html>
-<meta name="format-detection" content="telephone=no,date=no,address=no,email=no,url=no"/>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html data-editor-version="2" class="sg-campaigns" xmlns="http://www.w3.org/1999/xhtml">
 <head>
-  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700" rel="stylesheet">
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1, maximum-scale=1">
+  <!--[if !mso]><!-->
+  <meta http-equiv="X-UA-Compatible" content="IE=Edge">
+  <title>TestBoston password reset</title>
```

Comment:
Updated this email to use same HTML/styling as other emails. We missed this one because it is in Auth0 instead of in SendGrid.